src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/ReclusterServiceIntTest.kt

```diff
@@ -1047,6 +1048,12 @@ class ReclusterServiceIntTest : MessagingMultiNodeTestBase() {
 
       cluster2.checkReclusterMergeTelemetry(cluster1)
 
+      checkEventLog(personC.crn!!, CPRLogEvents.CPR_RECLUSTER_UUID_MERGED) { eventLogs ->
+        assertThat(eventLogs).hasSize(1)
+        val eventLog = eventLogs.first()
+        assertThat(eventLog.uuid).isEqualTo(cluster2.personUUID)
+        assertThat(eventLog.uuidStatusType).isEqualTo(UUIDStatusType.RECLUSTER_MERGE)
```

Comment:
maybe verify that the `from` entity is in this log as well? I'm sure it will be there - code looks fine